client/src/components/Offer/OfferDetails/OfferDetails.tsx

```diff
@@ -35,13 +30,11 @@ export const OfferDetails = ({
       <div className={styles.positionDescription}>
         {tags && (
           <div className={styles.tags}>
-            {tags.map((tag, index) => {
-              return (
-                <div key={index} className={styles.tag}>
-                  {tag}
-                </div>
-              );
-            })}
+            {tags.map((tag, index) => (
+              <div key={index} className={styles.tag}>
```

Comment:
I think we can assume that the tag is unique and use this as a key.
```suggestion
            {tags.map((tag) => (
              <div key={tag} className={styles.tag}>
```